cmd/main.go

```diff
@@ -18,26 +18,36 @@ package main
 
 import (
 	"log"
-
 	"sigs.k8s.io/kubebuilder/v3/pkg/cli"
 	cfgv2 "sigs.k8s.io/kubebuilder/v3/pkg/config/v2"
 	cfgv3 "sigs.k8s.io/kubebuilder/v3/pkg/config/v3"
+	"sigs.k8s.io/kubebuilder/v3/pkg/plugin"
+	commonv1 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/common/v1"
 	declarativev1 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/declarative/v1"
-	pluginv2 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/v2"
-	pluginv3 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/v3"
+	"sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang"
+	golangv2 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/v2"
+	golangv3 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/v3"
 )
 
 func main() {
+
+	// Bundle plugin which built the golang projects scaffold by Kubebuilder
+	gov3Bundle, _ := plugin.NewBundle(golang.DefaultGoNameQualifier, plugin.Version{Number: 3},
+		commonv1.Plugin{},
+		golangv3.Plugin{},
+	)
+
 	c, err := cli.New(
 		cli.WithCommandName("kubebuilder"),
 		cli.WithVersion(versionString()),
 		cli.WithPlugins(
-			&pluginv2.Plugin{},
-			&pluginv3.Plugin{},
+			&golangv2.Plugin{},
+			gov3Bundle,
+			&commonv1.Plugin{},
```

Comment:
We should probably also register the "base.go.kubebuilder.io" plugin so that when a different "common" plugin is implemented (for example including config-gen) we can provide `--plugins=common/v2,base.go/v3`:
```suggestion
			&golangv2.Plugin{},
			&golangv3.Plugin{},
			gov3Bundle,
			&commonv1.Plugin{},
```

Reply:
Let's leave that for when we will be able to use it with the config-gen. IHMO let's try to provide only the minimal set for what we are trying to use now. We can add that after since is not a breaking change and it is very easy too.

Reply:
PS.: I am just adding the base here for we are able to provide an option to create the common base and then scaffold the rest with any language and not KB lib.

Reply:
Could we agree on that?